src/api/db/data/20230411174510_add_loongarch64.rb

```diff
@@ -0,0 +1,9 @@
+class AddLoongarch64 < ActiveRecord::Migration[5.1]
```

Comment:
```suggestion
class AddLoongarch64 < ActiveRecord::Migration[7.0]
```
This seems copy-pasted from other data migrations. I tested the data migration generator (`bin/rails generate data_migration ...`). It returned `ActiveRecord::Migration[7.0]`.

Is there any reason to not using the now 7.0 version default?